doc/nrf/dm_code_base.rst

```diff
@@ -66,21 +66,23 @@ In this way, you can decide to work with a specific |NCS| release either by init
 Alternatively, you can work with the latest state of development by using the main branch of the `sdk-nrf`_ repository, updating it with Git regularly and using ``west update`` to update the project repositories every time the manifest repository changes.
 More information about manifests can be found in the :ref:`west manifest section <zephyr:west-manifests>` of the Zephyr documentation.
 
+.. _dm-revisions:
+
 Revisions
 *********
 
 There are two fundamental revisions that are relevant to most |NCS| users:
 
 * The ``main`` branch of the `sdk-nrf`_ repository
-* Any Git tag (that is, release) of the `sdk-nrf`_ repository
+* Any Git tag (that is, release, release candidate or development tag) of the `sdk-nrf`_ repository
 
 As discussed above, the revision of the manifest repository, `sdk-nrf`_, uniquely determines the revisions of all other repositories, so a discussion about |NCS| revisions can be essentially limited to the manifest repository revision.
 
 The ``main`` branch of the `sdk-nrf`_ repository always contains the latest development state of the |NCS|.
 Since all development is done openly, you can use it if you are not particularly concerned about stability and want to track the latest changes that are being merged continuously into the different repositories.
 
 The Git tags correspond to official releases tested and signed by the Nordic engineers.
-The format is as follows::
+The format for nRF repositories is as follows::
 
   vX.Y.Z(-rcN)
```

Comment:
```suggestion
vX.Y.Z(-rcN)
```

Reply:
https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#literal-blocks

Reply:
If leading space is removed, this won't be rendered as a literal block, see `::` above.